templates/event-translations-header.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Wporg\TranslationEvents;
+
+use GP;
+use Wporg\TranslationEvents\Event\Event;
+
+/** @var Event  $event */
+
+/* translators: %s: Event title. */
+gp_title( sprintf( __( 'Translation Events - %s' ), esc_html( $event->title() ) ) );
+gp_breadcrumb_translation_events( array( '<a href="' . esc_attr( gp_url( wp_make_link_relative( get_the_permalink( $event->id() ) ) ) ) . '">' . esc_html( $event->title() ) . '</a>', __( 'Translations', 'glotpress' ), $locale ) );
```

Comment:
Currently, we add the `english name` and the `locale name`.

![image](https://github.com/WordPress/wporg-gp-translation-events/assets/1667814/e7e1173f-c05b-40e3-a19b-975857537e48)

This update uses only the `english name`.

![image](https://github.com/WordPress/wporg-gp-translation-events/assets/1667814/317aaf8e-0295-4fc4-9867-4e8ed9bbab41)

```suggestion
gp_breadcrumb_translation_events( array( '<a href="' . esc_attr( gp_url( wp_make_link_relative( get_the_permalink( $event->id() ) ) ) ) . '">' . esc_html( $event->title() ) . '</a>', __( 'Translations', 'glotpress' ), $locale->english_name ) );
```